.github/workflows/wheel.yaml

```diff
@@ -0,0 +1,56 @@
+name: Python wheel
+
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
This seems to be triggered by regular commits on this PR, is that expected? Is this action going to publish draft packages first? Can we use a staging pypi repo to test this github actions, or run it with a dry-run option at first?

Reply:
This workflow is triggered when a new tag is pushed, not by every commit.  I am manually pushing tags for each commit for testing purposes.  We can also delete any botched release on PyPI, so I am not too worried about that.